src/components/Header/components/NavButtonsRightCorner/navButtons.js

```diff
@@ -192,67 +192,6 @@ export function createTimesheetsButton(networkStatus, authenticated, onLinkClick
   return timesheetsButton;
 }
 
-/**
```

Comment:
It was explained in the commit message, based on what Josh described during our video call meeting. It was decided (from discussions with the leaders of the 360 project) that we will make the ApartmentApp page only accessible via URL, since it is only needed at specific times (analogous to the ID Uploader).
So the link would be emailed out to people when the appropriate time of year comes around, just like the way the ID Uploader is used apparently